tracing-appender/src/rolling/compression.rs

```diff
@@ -0,0 +1,217 @@
+//! Defines configuration for passing compression options
+//!
+//! Currently only gzip compression is implemented.
+use flate2::Compression;
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub(crate) enum GzipCompressionLevelLiteral {
+    None,
+    Fast,
+    Best,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+#[repr(u32)]
```

Comment:
do we need `u32`? Seems we could use `u8` and that would be sufficient

Reply:
Fixed. Will push in a moment.